src/package-manager.js

```diff
@@ -419,14 +419,17 @@ module.exports = class PackageManager {
 
     for (const packageDirPath of this.packageDirPaths) {
       if (fs.isDirectorySync(packageDirPath)) {
-        for (let packagePath of fs.readdirSync(packageDirPath)) {
-          packagePath = path.join(packageDirPath, packagePath);
-          const packageName = path.basename(packagePath);
+        const packagePaths = fs
```

Comment:
```suggestion
        const packageNames = fs
```
I thinks the most appropriate name for this would be `packageNames`.